packages/xrpl/src/Wallet/rfc1751.ts

```diff
@@ -136,7 +136,13 @@ function getSubKey(
   const ch = [0, 0, 0, 0, 0, 0, 0, 0, 0]
   let word = ''
   for (word of sublist) {
-    const idx = rfc1751WordList.indexOf(word)
+    const idx = rfc1751WordList.indexOf(word.toUpperCase())
```

Comment:
Instead of allowing words to be lowercase (or have lowercase letters), it may make sense to throw an error. I believe rippled RFC-1751 always provides mnemonics in uppercase. If the mnemonic is not provided in uppercase, it is likely that the caller is doing something wrong.

If they're not doing something wrong, then it is trivial for them to use `toUpperCase()` themselves - which helps to ensure that they know what they are doing.

```suggestion
    const idx = rfc1751WordList.indexOf(word)
```